templates/_layout-no-footer.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
```

Comment:
Can you not use one base layout and add/or not the footer on the template, so we don't have to maintain 2 templates with duplicate html?